ui/src/app/utils/url.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Builds url type of string
+ * @param {string[]} commands ex - ['ui','td']
+ * @returns {string} string -- ex - '/ui/td/'
+ */
+
+export const buildUrl = (commands:string[]) => {
+  commands.unshift('');
+  commands.push('');
```

Comment:
this can be simplified with `['', ...commands, '']` @manikanta-reddy-t

Reply:
or `'/' + commands.join('/') + '/'` also may solve this, right?

Reply:
yes